mkosi/__init__.py

```diff
@@ -1159,14 +1107,13 @@ def load_args(args: argparse.Namespace) -> MkosiConfig:
     if args.cmdline and args.verb not in MKOSI_COMMANDS_CMDLINE:
         die(f"Parameters after verb are only accepted for {list_to_string(verb.name for verb in MKOSI_COMMANDS_CMDLINE)}.")
 
-    if args.bootable:
-        if args.verb == Verb.qemu and args.output_format in (
-            OutputFormat.directory,
-            OutputFormat.subvolume,
-            OutputFormat.tar,
-            OutputFormat.cpio,
-        ):
-            die("Directory, subvolume, tar, cpio, and plain squashfs images cannot be booted.", MkosiNotSupportedException)
+    if args.verb == Verb.qemu and args.output_format in (
+        OutputFormat.directory,
+        OutputFormat.subvolume,
+        OutputFormat.tar,
+        OutputFormat.cpio,
+    ):
+        die("Directory, subvolume, tar, cpio, and plain squashfs images cannot be booted.", MkosiNotSupportedException)
```

Comment:
This should mentioned Qemu (and should have earlier, too, probably), since at least a directory will work fine `systemd-nspawn -b`, no